test/e2e_node/jenkins/cri_validation/benchmark-config.yaml

```diff
@@ -1,56 +1,56 @@
 ---
 images:
   gci-density1:
-    image: gci-dev-55-8872-18-0
+    image: gci-dev-56-8977-0-0
```

Comment:
nit: we should may be make these tests derive from a common subsection to avoid updating the image name so many times. Not necessary for this PR though.